data/filter/BaseFilterFieldSpec.ts

```diff
@@ -165,6 +189,7 @@ export abstract class BaseFilterFieldSpec extends HoistBase {
     private getDefaultOperators(): FieldFilterOperator[] {
         if (this.isBoolFieldType) return ['='];
         if (this.isCollectionType) return ['includes', 'excludes'];
+        if (this.isMismatchedDateFieldType) return ['>', '>=', '<', '<='];
```

Comment:
This ensures the FilterChooser and GridFilter will offer the same operators for this field, which is not currently the case in the ActivityTracking example - only the GridFilter offers equals/does not equal.